src/Analysis/GWASV2/Steps/SelectCovariates/SelectCovariates.css

```diff
@@ -0,0 +1,57 @@
+.GWASV2 .GWASUI-double-column {
+  min-width: 988px;
+}
+.GWASV2 .select-covariates-container {
+  padding: 20px;
+  border: 1px solid #e2e2e3;
+  background: #fff;
+  border-radius: 4px;
+  height: 519px;
+}
+.GWASV2 .select-covariates-container .GWASUI-table1 + button,
```

Comment:
follow-up:
- delete css and have ticket to let covariates component accept children? So that we can pass in a chart? Then the chart could be positioned below the buttons?

Reply:
Ticket created: https://ctds-planx.atlassian.net/jira/software/projects/VADC/boards/132/backlog?selectedIssue=VADC-373